scripts/db/types/db-types.d.ts

```diff
@@ -1,18 +1,31 @@
-import { User } from 'auth0'
-
-/** Shape of user object that Database Action Scripts expect to be returned */
-interface CallbackUser extends User {
+/**
+ * Shape of user object that Database Action Scripts expect to be returned.
+ *
+ * There is no predefined type for this, we are using this documentation as
+ * a guide for what is allowed:
+ * https://auth0.com/docs/manage-users/user-accounts/user-profiles/normalized-user-profile-schema
+ */
+export interface CallbackUser {
   /**
    * This appears to be necessary, despite no documentation I could find in a
    * cursory inspection
    */
   id: string
+  email: string
+  email_verified?: boolean
+  nickname?: string
+  given_name?: string
+  family_name?: string
+  picture?: string
 }
 /** Signature of Database Action Script callback */
-type DbScriptCallback = (error: Error | null, person?: CallbackUser) => any
+export type DbScriptCallback = (
+  error: Error | null,
+  person?: CallbackUser
+) => any
 
 /** Configuration available from connection environment variables */
-interface DbConfiguration {
+export interface DbConfiguration {
```

Comment:
Required to prevent "db-types.d.ts is not a module" warning after removing the import at the top